assets/stylesheets/modules/_checkout.scss

```diff
@@ -209,6 +209,11 @@ $sprite-card-offset-visa:               -355px;
         margin-left: 35%;
     }
 }
+.review-details__notes {
+    @include fs-textSans(1);
+    line-height: 1.125rem;
```

Comment:
Use pixels since we have a px-to-rem postprocessor in our Grunt compile task which will turn these into rem